From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Thu, 25 Jan 2024 19:29:59 +0100
Subject: [PATCH] Load packs - Content - Items - Register - Call after vanilla
 bootstrap

This calls the item registration defined in the patch "Load packs - Content - Items - Register - Implementation"
with the items passed to the bootstrap in the patch "Load packs - Content - Definitions - Pass read definitions to bootstrap"
at the appropriate time after the vanilla items have been bootstrapped, but before any plugin bootstrap code is called.

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Fiddle - https://fiddlemc.org

diff --git a/src/main/java/net/minecraft/server/Bootstrap.java b/src/main/java/net/minecraft/server/Bootstrap.java
index fba0a71694b0e9f15e5424850970fcdd5d47b3eb..bfc59c5fdce837dc7ae4f4f49346a5617a0b849c 100644
--- a/src/main/java/net/minecraft/server/Bootstrap.java
+++ b/src/main/java/net/minecraft/server/Bootstrap.java
@@ -88,6 +88,7 @@ public class Bootstrap {
                     DispenseItemBehavior.bootStrap();
                     CauldronInteraction.bootStrap();
                     org.fiddlemc.fiddle.pack.load.fiddlecontent.block.BlockByDefinitionLoader.INSTANCE.register(readContentDefinitions.get(org.fiddlemc.fiddle.pack.read.fiddlecontent.definition.block.BlockDefinition.TYPE)); // Fiddle - load packs - blocks - register - call after vanilla bootstrap
+                    org.fiddlemc.fiddle.pack.load.fiddlecontent.item.ItemByDefinitionLoader.INSTANCE.register(readContentDefinitions.get(org.fiddlemc.fiddle.pack.read.fiddlecontent.definition.item.ItemDefinition.TYPE)); // Fiddle - load packs - items - register - call after vanilla bootstrap
                     // Paper start
                     BuiltInRegistries.bootStrap(() -> {
                         net.minecraft.core.Registry.register(net.minecraft.core.registries.BuiltInRegistries.MATERIAL_CONDITION, new net.minecraft.resources.ResourceLocation("paper", "bedrock_condition_source"), net.minecraft.data.worldgen.SurfaceRuleData.PaperBedrockConditionSource.CODEC.codec());
